### ISSUE TITLE
VF Mode: Tensor allocation limit causes operation failures

### ISSUE THREAD

**Issue by Hemaprasannakc** [torch-spyre/torch-spyre#446]
## Summary

VF mode has a tensor allocation limit that causes operations requiring more than ~3-4 tensors to fail with VF streaming errors. This is not specific to certain operations, but occurs when cumulative tensor allocations exceed VF capacity.

### Working Example (GELU operates within VF mode tensor limits:)

**Compilation output:**
```
(dti-venv) [root@ccadba2be78a examples]# python3 /tmp/test_suite_gelu_tanh.py
 ERRR 28.01.2026 06:29:02.106209 [                  ras_base.hpp:  94] {"MessageAppId":0,"MessageTag":0,"MessageType":19,"VfId":0,"action":"information","category":"software","code":"0x1d8c","description":"An unsupported message type was sent.","message":"An unsupported message type was sent.","name":"RAS::VFMSG::UnsupportedMessageType","severity":"ERROR","step":"Check program compatibility with firmware.","type":"runtime_error"}
/tmp/test_suite_gelu_tanh.py:8: UserWarning: Warning only once for all operators,  other operators may also be overridden.
  Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::view(Tensor(a) self, SymInt[] size) -> Tensor(a)
    registered at /home/pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: PrivateUse1
  previous kernel: registered at /home/pytorch/aten/src/ATen/LegacyBatchingRegistrations.cpp:1079
       new kernel: registered at /root/dt-inductor/torch-spyre/codegen/outputs/spyre_torch_ops.cpp:6213 (Triggered internally at /home/pytorch/aten/src/ATen/core/dispatch/OperatorEntry.cpp:218.)
  x_device = x.to(DEVICE)
Generating /tmp/torchinductor_root/inductor-spyre/sdsc_fused_gelu_0__swagr1r/execute/sdsc_fused_gelu_0/sdsc.json
RUN: sdsc_fused_gelu_0 /tmp/torchinductor_root/inductor-spyre/sdsc_fused_gelu_0__swagr1r/g2.graph.cbor
✓ GELU (tanh): delta=0.00146484375

```

### Failing Example (SiLU)

**Compilation output:**
```
(dti-venv) [root@ccadba2be78a _inductor]# pytest test_inductor_ops.py::TestOps::test_activation_fn_silu_fp16 -v -s
====================================================================== test session starts ======================================================================
platform linux -- Python 3.12.9, pytest-9.0.1, pluggy-1.6.0 -- /root/dt-inductor/dti-venv/bin/python3
cachedir: .pytest_cache
hypothesis profile 'default'
rootdir: /root/dt-inductor/torch-spyre
configfile: pyproject.toml
plugins: hypothesis-6.148.2
collecting ... [W128 16:15:17.282810903 OperatorEntry.cpp:218] Warning: Warning only once for all operators,  other operators may also be overridden.
  Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::view(Tensor(a) self, SymInt[] size) -> Tensor(a)
    registered at /home/pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: PrivateUse1
  previous kernel: registered at /home/pytorch/aten/src/ATen/LegacyBatchingRegistrations.cpp:1079
       new kernel: registered at /root/dt-inductor/torch-spyre/codegen/outputs/spyre_torch_ops.cpp:6213 (function operator())
 ERRR 28.01.2026 16:15:17.224571 [                  ras_base.hpp:  94] {"MessageAppId":0,"MessageTag":0,"MessageType":19,"VfId":0,"action":"information","category":"software","code":"0x1d8c","description":"An unsupported message type was sent.","message":"An unsupported message type was sent.","name":"RAS::VFMSG::UnsupportedMessageType","severity":"ERROR","step":"Check program compatibility with firmware.","type":"runtime_error"}
collected 1 item                                                                                                                                                

test_inductor_ops.py::TestOps::test_activation_fn_silu_fp16 Generating /tmp/torchinductor_root/inductor-spyre/sdsc_fused_silu_0_8qaedata/execute/sdsc_fused_silu_0/sdsc.json
Generating /tmp/torchinductor_root/inductor-spyre/sdsc_fused_silu_1_ux2xtpsn/execute/sdsc_fused_silu_1/sdsc.json
RUN: sdsc_fused_silu_0 /tmp/torchinductor_root/inductor-spyre/sdsc_fused_silu_0_8qaedata/g2.graph.cbor
RUN: sdsc_fused_silu_1 /tmp/torchinductor_root/inductor-spyre/sdsc_fused_silu_1_ux2xtpsn/g2.graph.cbor
 ERRR 28.01.2026 16:15:22.933368 [               vf_streamer.cpp: 417] Detected 1 bad response blocks in message stream
terminate called after throwing an instance of 'std::runtime_error'
  what():  Bad responses and/or bad response blocks detected
Signal Received: 6 (Aborted)
Signal Received from pid=377 
*** BACKTRACE ***
/opt/ibm/spyre/senlib/lib/libsenlib-dd2.so(+0x2c788e)[0x3ff9c24788e]
linux-vdso64.so.1(__kernel_rt_sigreturn+0x0)[0x3ffc02b1490]
/lib64/libc.so.6(+0x98c26)[0x3ffbd218c26]
/lib64/libc.so.6(gsignal+0

```

## Error Details

**Primary Error:**
```
File: vf_streamer.cpp:417
Error: "Detected 1 bad response blocks in message stream"
Type: VF streaming failure during kernel execution
```

**Additional Warning:**
```
RAS::VFMSG::UnsupportedMessageType
MessageType: 19
Code: 0x1d8c
Message: "An unsupported message type was sent"
```

### What did you expect to happen?

All compiled operations should execute successfully in VF mode, regardless of intermediate tensor requirements. Operations requiring more storage (SiLU, softmax, broadcasting) should work without streaming errors.

### How can we reproduce it?

Running pytest test case given in [inductor_ops_test_suite](https://github.com/torch-spyre/torch-spyre/blob/main/tests/_inductor/test_inductor_ops.py)

```
pytest test_inductor_ops.py::TestOps::test_activation_fn_silu_fp16 -v -s
```

## Environment
- **Platform**: IBM Z (s390x),
- **Device Mode**: VF (Virtual Function)
- **Configuration**: DOOM enabled, ENABLE_RISCV=0x0
- **Firmware**: Reports MessageType 19 as unsupported
- **Build**: 
  - torch-spyre: main branch
  - senlib: 1.2.0
  - PyTorch: 2.9.1

### Relevant log output

```
(dti-venv) [root@ccadba2be78a _inductor]# pytest test_inductor_ops.py::TestOps::test_add_broadcast_cpu_256_67x256 -v -s
====================================================================== test session starts ======================================================================
platform linux -- Python 3.12.9, pytest-9.0.1, pluggy-1.6.0 -- /root/dt-inductor/dti-venv/bin/python3
cachedir: .pytest_cache
hypothesis profile 'default'
rootdir: /root/dt-inductor/torch-spyre
configfile: pyproject.toml
plugins: hypothesis-6.148.2
collecting ... [W128 16:29:51.929848798 OperatorEntry.cpp:218] Warning: Warning only once for all operators,  other operators may also be overridden.
  Overriding a previously registered kernel for the same operator and the same dispatch key
  operator: aten::view(Tensor(a) self, SymInt[] size) -> Tensor(a)
    registered at /home/pytorch/build/aten/src/ATen/RegisterSchema.cpp:6
  dispatch key: PrivateUse1
  previous kernel: registered at /home/pytorch/aten/src/ATen/LegacyBatchingRegistrations.cpp:1079
       new kernel: registered at /root/dt-inductor/torch-spyre/codegen/outputs/spyre_torch_ops.cpp:6213 (function operator())
 ERRR 28.01.2026 16:29:51.874282 [                  ras_base.hpp:  94] {"MessageAppId":0,"MessageTag":0,"MessageType":19,"VfId":0,"action":"information","category":"software","code":"0x1d8c","description":"An unsupported message type was sent.","message":"An unsupported message type was sent.","name":"RAS::VFMSG::UnsupportedMessageType","severity":"ERROR","step":"Check program compatibility with firmware.","type":"runtime_error"}
collected 1 item                                                                                                                                                

test_inductor_ops.py::TestOps::test_add_broadcast_cpu_256_67x256 Generating /tmp/torchinductor_root/inductor-spyre/sdsc_fused_add_unsqueeze_0_v0b0js51/execute/sdsc_fused_add_unsqueeze_0/sdsc.json
RUN: sdsc_fused_add_unsqueeze_0 /tmp/torchinductor_root/inductor-spyre/sdsc_fused_add_unsqueeze_0_v0b0js51/g2.graph.cbor
 ERRR 28.01.2026 16:29:57.615220 [               vf_streamer.cpp: 417] Detected 1 bad response blocks in message stream
terminate called after throwing an instance of 'std::runtime_error'
  what():  Bad responses and/or bad response blocks detected
Signal Received: 6 (Aborted)
Signal Received from pid=769 
*** BACKTRACE ***


```
